torch/csrc/stable/ops.h

```diff
@@ -215,4 +217,15 @@ inline Tensor copy_(
   return to<Tensor>(stack[0]);
 }
 
+// We expect this to be the stable version of the clone op.
+inline Tensor clone(
+    const Tensor& self,
+    std::optional<c10::MemoryFormat> memory_format = std::nullopt) {
```

Comment:
Let's not support the optional kwarg just yet

Reply:
Optional memory_format support is now removed.